api/src/pcapi/routes/backoffice/autocomplete.py

```diff
@@ -76,6 +77,66 @@ def autocomplete_offerers() -> AutocompleteResponse:
     )
 
 
+def _get_institution_choice_label(institution: educational_models.EducationalInstitution) -> str:
+    return f"{institution.institutionType} {institution.name.strip()} - {institution.city}"
+
+
+def _get_institutions_base_query() -> sa.orm.Query:
+    return educational_models.EducationalInstitution.query.options(
+        sa.orm.load_only(
+            educational_models.EducationalInstitution.id,
+            educational_models.EducationalInstitution.name,
+            educational_models.EducationalInstitution.institutionType,
+            educational_models.EducationalInstitution.city,
+        )
+    )
+
+
+def prefill_institutions_choices(autocomplete_field: fields.PCTomSelectField) -> None:
+    if autocomplete_field.data:
+        institutions = (
+            _get_institutions_base_query()
+            .filter(educational_models.EducationalInstitution.id.in_(autocomplete_field.data))
+            .order_by(
+                educational_models.EducationalInstitution.institutionType,
+                educational_models.EducationalInstitution.name,
+            )
+        )
+        autocomplete_field.choices = [
+            (institution.id, _get_institution_choice_label(institution)) for institution in institutions
+        ]
+
+
+@blueprint.backoffice_web.route("/autocomplete/institutions", methods=["GET"])
+@spectree_serialize(response_model=AutocompleteResponse, api=blueprint.backoffice_web_schema)
+def autocomplete_institutions() -> AutocompleteResponse:
+    query_string = request.args.get("q", "").strip()
+
+    is_numeric_query = query_string.isnumeric()
+    if not is_numeric_query and len(query_string) < 2:
+        return AutocompleteResponse(items=[])
+
+    if is_numeric_query:
+        filters = educational_models.EducationalInstitution.id == int(query_string)
+    else:
+        or_filters = [
+            sa.func.unaccent(educational_models.EducationalInstitution.name).ilike(f"%{clean_accents(query_string)}%"),
+            sa.func.unaccent(educational_models.EducationalInstitution.institutionType).ilike(
+                f"%{clean_accents(query_string)}%"
+            ),
+            sa.func.unaccent(educational_models.EducationalInstitution.city).ilike(f"%{clean_accents(query_string)}%"),
+        ]
+    filters = sa.or_(*or_filters)
```

Comment:
Un test montre une mauvaise indentation : 
```suggestion
        filters = sa.or_(*or_filters)
```
(mais ça va changer si tu cherches sur la concaténation indexée)